### PR TITLE
UNITY-8266 Add Product Data Sheet Link to Cart Resource

### PR DESCRIPTION
**Pull request by bolsu**
UNITY-8266

### CODE REVIEW

**Comment by bolsu** [comment]
@dkluever22 Hi again, another change to the API FYI. Thanks!

File: apps/data-types.md [hunk 1]
```diff
@@ -641,6 +641,7 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | energyLabelsString | string | A list of energy labels applied to this product. Can be one value or a range with two values. |
 | energyLabelsMaxEfficiencyString | string | The highest possible efficiency class of an uploaded energy label. |
 | energyLabelSourceFile | string | A URL with an image or PDF file containing the energy label image supplied by the manufacturer. |
+| productDataSheetSourceFile | string | A URL with an image or PDF file containing the data sheet for the product. |
```
Comment (dkluever22) [inline apps/data-types.md:644]: @bolsu Since `energyLabelSourceFile` is also listed under ##product and ##productSearch, shouldn't `productDataSheetSourceFile` be mentioned there as well? Or will this possibly be done with another PR?

**Review by dkluever22 — CHANGES_REQUESTED** [review]
@bolsu I left you two comments :) Furthermore I think it's necessary to add `productSheetDataSourceFile` to at least the events ### cart:add and ### cart:setQuantity depending on how things are planned regarding the product resource.
Please take a look :) Thank you in advance! 🙏

File: apps/data-types.md (hunk 1 above)
Comment (dkluever22) [inline apps/data-types.md:644]: To make the description consistent with `productDataSheet` within the product resources:
```suggestion
| productDataSheetSourceFile | string | A URL with an image or PDF file containing a data sheet with technical information on the product. Has to be available if the product has an energy label. |
```

Comment (bolsu) [reply apps/data-types.md:644]: 'Has to be available if the product has an energy label.'

I see what you mean, but this isn't really true, the productDataSheet is independent of the energy label stuff. Perhaps this quote was meant for 'energyLabelSourceFile' instead, that would actually make sense.

Comment (dkluever22) [reply apps/data-types.md]: @bolsu I see. But maybe we should then correct it and also add this information to the description of `productDataSheetSourceFile` for the data type productLineItem? So something around these lines: "Has to be available if `energyLabelSourceFile` is provided."
What do you think?

Comment (bolsu) [reply apps/data-types.md]: @dkluever22 That line is incorrect for both Product / ProductLineItem, it should not be there for either. Have I misunderstood something?

Comment (dkluever22) [reply apps/data-types.md]: @bolsu No, it seems that it was a misunderstanding on our end. So you're saying that in the description of `productDataSheet` the statement "Has to be available if the product has an energy label." is incorrect. The only place where this sentence makes sense is within the description of `energyLabelSourceFile`. Correct?
So no dependencies between product data sheets and energy labels?
I'm just wondering why this information was put there in the first place and if it had anything to do with this ticket: https://epages.atlassian.net/browse/EPUI-1988

Comment (bolsu) [reply apps/data-types.md]: @dkluever22 Now we're on the same page :)

I don't think it is to do with that issue, it was probably done a long time ago. We could probably find out if we cared enough to go through the file's history ...

Comment (dkluever22) [reply apps/data-types.md]: @bolsu Oooh, I see. Maybe when we will have a loooot of time we can check it out :D
